config/production/prometheus.yml

```
@@ -0,0 +1,109 @@
+      - targets: ['sqlite-exporter:9092']
+    metrics_path: /metrics
+    scrape_interval: 60s
+
+# Recording rules for aggregated metrics
+recording_rules:
+  - name: church_management_rules
+    rules:
+      # Application response time percentiles
+      - record: church_management:http_request_duration_seconds:p95
+        expr: histogram_quantile(0.95, sum(rate(http_request_duration_seconds_bucket[5m])) by (le, job, instance))
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)